src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx

```diff
@@ -35,14 +37,20 @@ class GroupTags extends React.Component {
     this.fetchData();
   }
 
+  componentDidUpdate(prevProps) {
+    if (!isEqual(prevProps.environments, this.props.environments)) {
+      this.fetchData();
+    }
+  }
+
   fetchData = () => {
-    const {api, group} = this.props;
+    const {api, group, environments} = this.props;
+    const env_query = environments.map(e => '&environment=' + e).join('');
```

Comment:
nit: use `camelCase`

Reply:
I think more correct would be 
```
environments.map(e => 'environment=' + e).join('&')
```
So you don't have the leading `&`

But I think even better would be to not stringify URL parameters at all and pass them in the 2nd parameter of `api.request` using the `data` key.

Reply:
Thanks Billy. Will note the camelCase preference. Removed the stringify'd parameters in favour of using the query key.